custom/ilsgateway/tanzania/reports/facility_details.py

```diff
@@ -176,6 +176,7 @@ def headers(self):
     def rows(self):
         data = (SMS.by_domain(self.config['domain'])
                 .filter(location_id=self.config['location_id'])
+                .exclude(processed=False)
```

Comment:
we should only exclude outgoing messages that weren't processed, but include incoming messages whether they were processed or not. so we can just add direction=OUTBOUND to this line